### PR TITLE
Adds login page validation

### PR DESCRIPTION
**Pull request by thiessenp-cds**
# Summary | Résumé

Adds login page validation

### CODE REVIEW

File: cypress/integration/login_page.spec.js [hunk 1]
```diff
@@ -21,32 +23,74 @@ describe("Login Page", () => {
...
 
-    it("Displays an error message when submitting a form and getting an unsuccessful reply from the server.", () => {
-      cy.intercept("POST", "/api/token/temporary", {
-        statusCode: 403,
-      }).as("userSuccess");
-      cy.get("input[id='loginEmail']").type("test@cds-snc.ca");
+    it("Displays an error message when submitting an empty username.", () => {
       cy.get("[type='submit']").click();
-      cy.get("[data-testid='alert']").should("be.visible");
+      cy.get("[id='errorMessageusername']").should(
```
Comment (thiessenp-cds) [inline cypress/integration/login_page.spec.js:28]: The login validation tests could also go outside of the Bearer Token State tests

File: cypress/integration/login_page.spec.js [hunk 2]
```diff
@@ -21,32 +23,74 @@ describe("Login Page", () => {
...
+    it("Displays an error message when submitting a form and getting an unsuccessful reply from the server.", () => {
+      cy.intercept("POST", "/api/token/temporary", {
+        statusCode: 403,
+      }).as("userSuccess");
+      cy.get("input[id='username']").type("test@cds-snc.ca");
       cy.get("[type='submit']").click();
       cy.get("[data-testid='alert']").should("be.visible");
     });
   });
+
```
Comment (thiessenp-cds) [inline cypress/integration/login_page.spec.js:76]: This test was failing (no loginEmail ID or signInKy ID in the page). I thought it best to comment out for now - even though this is not a BP :)

File: cypress/integration/login_page.spec.js (hunk 1 above)
Comment (Moro-Code) [reply cypress/integration/login_page.spec.js:28]: I will remove these tests as they no longer apply

File: cypress/integration/login_page.spec.js (hunk 2 above)
Comment (Moro-Code) [reply cypress/integration/login_page.spec.js:76]: Yup these are old tests. Going to remove for now as they no longer apply

**Review by Moro-Code — APPROVED** [review]

**Pull request merged**